src/mono/mono/metadata/marshal-lightweight.c

```diff
@@ -662,7 +663,8 @@ gc_safe_transition_builder_init (GCSafeTransitionBuilder *builder, MonoMethodBui
 	builder->coop_cominterop_fnptr = -1;
 #endif
 #if defined (TARGET_WASM)
-	return FALSE;
```

Comment:
Shouldn't there be safepoints also at runtime for interp code ? Seems like this is only adding them for the aot compiler.

Reply:
Yeah, but the interp does them in a different spot, I think?

Reply:
The interpreter does the gc safe transitions only if the wrapper does them. So I think the correct condition is to add them is if `mono_opt_wasm_gc_safepoints` or `ifndef DISABLE_THREADS`

Reply:
Aha, that makes sense. My first swing at this changed it to TRUE.